.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: python
 env:
-  - DJANGO="22"
+  - DJANGO="Django>=2.2,<=3.0"
```

Comment:
Shouldn't it be
```yml
    DJANGO="Django>=2.2,<3.0"
```
?

Reply:
Now you get Django 3.0.

Reply:
Good shout, brain dead from Friday afternoon!